.vib/jenkins/goss/jenkins.yaml

```diff
@@ -0,0 +1,16 @@
+command:
+  check-app-jar:
+    exec: java -jar /opt/bitnami/jenkins/jenkins-plugin-manager.jar
+    exit-status: 0
+  check-app-version:
+    exec: java -jar /opt/bitnami/jenkins/jenkins.war --version
+    exit-status: 0
+    stdout:              
+      - {{ .Env.APP_VERSION }}
```

Comment:
I think you can safely remove this and use the `check-app-version.yaml` test instead, with `java -jar /opt/bitnami/jenkins/jenkins.war` as the command